### ISSUE TITLE
CV Comparison

### ISSUE THREAD

**Issue by peterbomberg** [hres/pmp#73]
Missing the CV list for Units of Measure.

Missing the definitions for Route of Administration, Dosage Form, Units of Measure (not sure what else is in the application as it's not available at this moment)

Potentially all remaining CV's identified in Annex A of the CV Harmonization presentation, such as Ingredients (active and non-active), shape, sex, colour, ....

**Comment by MartinBernard** [comment]
As discussed, we'll retrieve on Monday.  

RoA and DF list and definitions can also be found here drupal.hres.ca/vocabulary

However, it can only be accessed within HC Intranet network.

The most of remaining CVs identified are not standardized in DPD.

**Comment by peterbomberg** [comment]
[PM CV Comparison.v2.xlsx](https://github.com/hres/pmp/files/101313/PM.CV.Comparison.v2.xlsx)

uploaded the current version

**Comment by peterbomberg** [comment]
Martin, can you get me the DPD units of measurement. I have attached the version showing where clarifications are required.
[PM CV Comparison.v4.xlsx](https://github.com/hres/pmp/files/103456/PM.CV.Comparison.v4.xlsx)

**Comment by MartinBernard** [comment]
@peterbomberg - Thanks Peter. What's our plan for getting the FDA definitions?. 

Retrieving the Metric Unit for you to add to your mapping exercise.

**Comment by peterbomberg** [comment]
I will be asking Veda on our next call, however there may be merit to
asking for them sooner.

**Comment by MartinBernard** [comment]
Let's ask now otherwise we wait another 10 days before the next call. Can you craft something? I'm in the process of draft an email to the group to share the mapping

**Comment by peterbomberg** [comment]
@vksrivas - Peter and I would like you to send an email to Veda before our meeting next week. See inbox for wording but it goes like this....

Veda,

Would you be able to assist with either putting us in contact with the point of contact for the definitions for the CV’s (Route of Administration, Dosage, Units of Measure, etc.) or provide the definitions for the CV’s.

**Comment by MartinBernard** [comment]
Email also sent to OSIP to cover the following : 

---

Hi All,

As a follow-up to our meeting last Monday, RMOD had committed in doing a cross walk mapping of some of the controlled vocabulary to be standardised. 

As promised, please find attached a preliminary version of the mapping. We started by listing all RoA and Dosage (see first two tabs) with their corresponding ID Code, HC Terms, HC Definitions with, what we think is, the FDA equivalent.  

You'll clearly notice that many of rows (highlighted in yellow) need verification, validation, and/or clarification. Some issues range from missing ID, to terms that are currently not being used and therefore should be inactivated.  

Bottom line, we will need to do some cleaning up before we publish these two lists as recognised controlled vocabularies.

Would it be possible to review and provide some initial input before we present at OSIP-MC this upcoming Friday. 

Please also note that a revised version of the deck presented at our meeting will be sent to OSIP-MC secreteriat by cob Wednesday.

**Comment by MartinBernard** [comment]
@peterbomberg - pls upload the CV harmonization deck here after my comments from this evening have been addressed.

**Comment by peterbomberg** [comment]
Revised version

[CV Harminization.v0.11.zip](https://github.com/hres/pmp/files/104417/CV.Harminization.v0.11.zip)

**Comment by MartinBernard** [comment]
@peterbomberg / @vksrivas - much better. I've included in the meeting invite for review tomorrow.

**Comment by MartinBernard** [comment]
@peterbomberg - Units of Measures Code Table has been sent to your personal email. I will also ask Eric to input into drupal.hres.ca/vocabulary

**Comment by MartinBernard** [comment]
Yes I'll set that up today following our osip-mc presentation. 

Can you please take this revised document and incorporate the units of measure mapping in here. Use this copy as version of the truth

Yes I'll set that up today following our osip-mc presentation. 

Can you please take this revised document and incorporate the units of measure mapping in here. Use this copy as version of the truth.

[Controlled Vocabulary (DF) January 2016.xlsx](https://github.com/hres/pmp/files/109673/Controlled.Vocabulary.DF.January.2016.xlsx)

**Comment by peterbomberg** [comment]
The units of measure were already incorporated in the version I send out
last time, I will update after disusing how to address the issues.

```
          Peter Bomberg
```

email: pbx@soundfound.com

cell: +1 613.769.1142

_From:_ MartinBernard [mailto:notifications@github.com]
_Sent:_ Friday, January 29, 2016 7:01 AM
_To:_ hres/pmp
_Cc:_ peterbomberg
_Subject:_ Re: [pmp] CV Comparison (#73)

Yes I'll set that up today following our osip-mc presentation.

Can you please take this revised document and incorporate the units of
measure mapping in here. Use this copy as version of the truth

Yes I'll set that up today following our osip-mc presentation.

Can you please take this revised document and incorporate the units of
measure mapping in here. Use this copy as version of the truth.

Controlled Vocabulary (DF) January 2016.xlsx
https://github.com/hres/pmp/files/109673/Controlled.Vocabulary.DF.January.2016.xlsx

**Comment by MartinBernard** [comment]
Please don’t incorporate at this time. We need to talk about how to resolve the issues and what the resolution means.  
[PM CV Comparison.v6.xlsx](https://github.com/hres/pmp/files/109820/PM.CV.Comparison.v6.xlsx)
I have attached the latest version.
